yktracec/src/ykllvmwrap.cc

```diff
@@ -461,13 +461,6 @@ void *compileIRTrace(FN Func, char *FuncNames[], size_t BBs[], size_t TraceLen,
     rewriteDebugInfo(JITMod, TraceName, DebugInfoFD,
                      filesystem::path(DebugInfoPath));
 
-#ifdef YK_TESTING
```

Comment:
A bit above, you should be able to kill `template <typename FN>` since there is now only one viable function to call.

Reply:
I plan to have a deeper look at that in a follow-up PR, although I could do it as a commit here if you'd like?

Reply:
as you wish.